docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -1243,6 +1243,22 @@ window.document.addEventListener('DOMContentLoaded', () => {
   );
 });
 ```
+[hint]
+GraphQL scaffolding derives the names for the schema from the first part of the namespace and the classname, while in
+the Injector the database table name is used. So in the case of
+```php
+namespace Foo\Bar;
+// …
+class MyVersionedObject extends DataObject
+{
+    private static $table_name = 'FBVersionedObject';
+    // …
+}
+```
+you would have to use `readOneFooMyVersionedObject`, `rollbackFooMyVersionedObject` and
```

Comment:
`readOneFooBarMyVersionedObject`? (seems like it's missing Bar)? Or is that intentional?

Reply:
It is intentional. And one of the reasons why it cost me so much time to get the example working in my setup. And that you are having to ask is even more prove that the docs need that kind of hint :-)
The graphQL scaffolding only uses the first part of the namespace  - see https://github.com/silverstripe/silverstripe-graphql/blob/0e44b8975ece50e448ebe39be499b40510f09b43/src/Scaffolding/StaticSchema.php#L96